src/main/java/ActionHandler.java

```diff
@@ -0,0 +1,66 @@
+import java.util.NoSuchElementException;
+import java.util.Scanner;
+
+
+public class ActionHandler {
+    enum allowedTask { todo , event , deadline};
+    /**
+     * Returns an action determined by the String command. By default, if the command is not any of the keywords
+     * returns a newTaskAction
+     * @param input specifies the type of action to return
+     * @return Action
+     */
+    public Action decideAction(String input) {
+        Scanner sc = new Scanner(input);
+        String command;
+        try{
+            command = sc.next();
+        } catch (NoSuchElementException e){
+            return new EmptyAction();
+        }
+
+        Action myAction = new EmptyAction();
+        switch(command){
```

Comment:
The spacing of this switch block is a little off. It should follow the structure as shown in the image i have attached.
<img width="1357" alt="Screenshot 2020-02-05 at 1 20 12 PM" src="https://user-images.githubusercontent.com/59982275/73813235-5758ac00-481a-11ea-9b76-017809085ef9.png">